builtin/rebase--helper.c

```diff
@@ -49,6 +47,9 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 			N_("insert exec commands in todo list"), ADD_EXEC),
 		OPT_CMDMODE(0, "append-todo-help", &command,
 			    N_("insert the help in the todo list"), APPEND_TODO_HELP),
+		OPT_CMDMODE(0, "edit-todo", &command,
```

Comment:
In the commit message you say "a new flag, `edit-todo`, is added" but it looks like the "flag" is a command mode so "a new command mode, `--edit-todo`, is added" might be better.